### ISSUE TITLE
Add keybindings to chat edit actions, help dialog

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#237649]
The following should have default keybindings to aid screen reader users and keyboard only users and be included in the a11y help dialog:

- Undo Last Edit (proposed `ctrlCmd+z`)
- Redo Last Edit (proposed `ctrlCmd+shift+z`)
- Clear Working Set (proposed `ctrlCmd+k`)
- Save All (proposed `alt+ctrlCmd+s`)
- View All Edits (proposed `ctrlCmd+e`)
- Add files (proposed `ctrlCmd+a`)
- Switch to Chat (already has keybinding)
- New Edit Session (already has keybinding)

cc @joyceerhl, @jrieken, @jooyoungseo, @rperez030 for feedback

**Comment by meganrogge** [comment]
also @isidorn

**Comment by isidorn** [comment]
While it makes sense to have default keyboard shortcuts for popular actions, I would not assign it to all.
Can we check telemetry to see what are actually the most popular of those actions? And then assign default keybindings for those.

I think that it is fine to make Copilot Edits GA without this, so feel free to push out to February if we do not find a consensus on the keybindings.

**Comment by meganrogge** [comment]
Most of these have keybindings now. Let's get user and tester feedback before assigining more.
